Classes/Settings/SettingsController.m

```diff
@@ -1,472 +0,0 @@
-                    
-                    [cell addConstraint:[NSLayoutConstraint constraintWithItem:encryptionKey attribute:NSLayoutAttributeTrailing relatedBy:NSLayoutRelationEqual toItem:cell.detailTextLabel attribute:NSLayoutAttributeTrailing multiplier:1 constant:0]];
-                    
-                    encryptionKey.textAlignment = NSTextAlignmentRight;
-                    
-                    encryptionKey.delegate = self;
-                    
-                    
-                } else {
-                    encryptionKey = (UITextField*)[cell.contentView viewWithTag:1];
-                }
-                
-                [encryptionKey addTarget:self action:@selector(encryptionPasswordChanged:) forControlEvents:(UIControlEventValueChanged | UIControlEventEditingDidEnd)];
-                [encryptionKey setKeyboardType:UIKeyboardTypeDefault];
-                [encryptionKey setDelegate:self];
-                [encryptionKey setSecureTextEntry:YES];
-                [encryptionKey setTag:1];
-                [[cell textLabel] setText:@"Password"];
-                [encryptionKey setPlaceholder:@"Enter Password"];
-                encryptionKey.text = [[Settings instance] encryptionPassword];
-                return cell;
-            }
-            default:
-                break;
-        }
-        
-    } else if (indexPath.section == CreditsGroup) {
-        static NSString *CellIdentifier = @"SettingsCreditsCell";
-        
-        UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:CellIdentifier];
-        if (cell == nil) {
-            cell = [[[UITableViewCell alloc] initWithStyle:UITableViewCellStyleSubtitle reuseIdentifier:CellIdentifier] autorelease];
-        }
-        
-        // Set up the cell...
-        if ([cell respondsToSelector:@selector(setSeparatorInset:)]) {
-            [cell setSeparatorInset:UIEdgeInsetsZero];
-        }
-        
-        [cell setSelectionStyle:UITableViewCellSelectionStyleNone];
-        
-        switch (indexPath.row) {
-            case 0:
-                [[cell textLabel] setText:@"Richard Moreland"];
-                [[cell detailTextLabel] setText:@"Original Author"];
-                break;
-            case 1:
-                [[cell textLabel] setText:@"Sean Escriva"];
-                [[cell detailTextLabel] setText:@"Development and Organization"];
-                break;
-            case 2:
-                [[cell textLabel] setText:@"Alex Rodich"];
-                [[cell detailTextLabel] setText:@"Development and Organization"];
-                break;
-            case 3:
-                [[cell textLabel] setText:@"Carsten Dominik"];
-                [[cell detailTextLabel] setText:@"Design and Emacs integration"];
-                break;
-            case 4:
-                [[cell textLabel] setText:@"Greg Newman"];
-                [[cell detailTextLabel] setText:@"Updated app icon"];
-                break;
-            case 5:
```

Comment:
This is an quite impressive list. If we would add the latest contributors it would become bigger again. On the old version it was limited to 5 entries for some reason. I would suggest to limit it again to 5 with recent contributors added. And a link to documentation where all contributors should be listed.